.github/workflows/java-build-for-release.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           name: project-release-artifacts
           path: ./release/
       - name: Create draft release
-        uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # v0.1.15
+        uses: softprops/action-gh-release@d99959edae48b5ffffd7b00da66dcdb0a33a52ee # v0.1.15
```

Comment:
Hmm, this tag comment isn't getting bumped.

Reply:
```suggestion
        uses: softprops/action-gh-release@d99959edae48b5ffffd7b00da66dcdb0a33a52ee # v2.0.2
```

Reply:
Yea, I don't know if this is a bug or intentional from dependabot, but if there's any deviation from the version number, it simply won't update it. i'd love a script that does a pass over all dependencies that dependabot can manage and fixes the comments (or for dependabot to just...do that)